game/state/city/economyinfo.cpp

```diff
@@ -35,7 +35,7 @@ bool EconomyInfo::update(GameState &state, bool xcom)
 		// Price update
 		if (soldThisWeek > 2 * maxStock)
 		{
-			currentPrice = currentPrice * randBoundsInclusive(state.rng, 85, 5);
+			currentPrice = currentPrice * randBoundsInclusive(state.rng, 85, 95);
```

Comment:
These values really shouldn't be added to the code - instead maybe make it somewhere in the GameState so it can be changed with a mod?